libs/user-monitoring/src/lib/user-monitoring.ts

```diff
@@ -20,8 +20,8 @@ const initDdRum = (params: DdRumInitParams) => {
     version: params.version,
     sampleRate: 10,
     sessionReplaySampleRate: 0,
-    trackInteractions: false,
-    allowedTracingOrigins: [
+    trackInteractions: true,
```

Comment:
What is the reason for this change? We recently changed it to false for security reasons.